account/account.go

```diff
@@ -71,10 +71,10 @@ type UpdateInput struct {
 // UpdateAccount updates your account details with the given parameters.
 // TODO(jen20) Work out a safe way to test this
 func (c AccountClient) Update(ctx context.Context, input *UpdateInput) (*Account, error) {
-	path := fmt.Sprintf("/%s", c.Client.AccountName)
+	fullPath := path.Join("/", c.Client.AccountName)
 	reqInputs := client.RequestInput{
 		Method: http.MethodPut,
-		Path:   path,
+		Path:   fullPath,
```

Comment:
The `respReader != nil` check below can be moved.  It doesn't have to be part of this PR, however.